File: typescript/05_functions.ts
function greet(name: string): string {
  return `hello ${name}`;
}

function add(a: number, b: number): number {
  return a + b;
}

const say = (msg: string = "hi"): void => {
  console.log(msg);
};

console.log(greet("ulas"));
console.log(add(3, 5));
say();
say("yo");

File: rust/05_functions.rs
fn greet(name: &str) -> String {
    format!("hello {}", name)
}

fn add(a: i32, b: i32) -> i32 {
    a + b
}

fn say(msg: &str) {
    println!("{}", msg);
}

fn main() {
    println!("{}", greet("ulas"));
    println!("{}", add(3, 5));
    say("hi");
    say("yo");
}
